sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/application-restriction-bypass.aest.controller.ts

```diff
@@ -24,16 +25,27 @@ export class ApplicationRestrictionBypassAESTController extends BaseController {
   }
 
   /**
-   * Get restriction bypasses for a given application.
+   * Get application restriction bypasses for a given application.
    * @param applicationId id of the application to retrieve restriction bypasses.
-   * @returns Application restriction bypasses.
+   * @returns application restriction bypasses.
    */
   @Get("application/:applicationId")
   async getApplicationRestrictionBypasses(
     @Param("applicationId", ParseIntPipe) applicationId: number,
-  ): Promise<ApplicationRestrictionBypassSummaryAPIOutDTO[]> {
-    return this.applicationRestrictionBypassService.getApplicationRestrictionBypasses(
-      applicationId,
+  ): Promise<ApplicationRestrictionBypassHistoryAPIOutDTO> {
+    const applicationRestrictionBypasses =
+      await this.applicationRestrictionBypassService.getApplicationRestrictionBypasses(
+        applicationId,
+      );
+    const bypasses = applicationRestrictionBypasses.map(
+      (item: ApplicationRestrictionBypass) => ({
+        id: item.id,
+        restrictionType: item.studentRestriction.restriction.restrictionType,
+        restrictionCode: item.studentRestriction.restriction.restrictionCode,
+        isActive: item.studentRestriction.isActive,
```

Comment:
Should we use `isRestrictionActive` instead of just `isActive` to avoid confusion? @dheepak-aot  any suggestion for this?

I am requesting this because it is a part of `ApplicationRestrictionBypassSummary`. One can confuse it with the `isRestrictionBypassActive`.